### ISSUE TITLE
BSOD: CLOCK_WATCHDOG_TIMEOUT

### ISSUE THREAD

**Issue by Elliander** [StrangeLoopGames/EcoIssues#7203]
On Windows 10, while running the latest version of the game, I will randomly experience a blue screen crash. The first time it happened was the last time I played on this computer 2 years ago. That crash corrupted a world, which I then reported and then stopped playing until now. This time, instead of running my own server, I'm playing in another. The first time it happened with the latest build I was actively mining stone, however there were other programs running at the same time. The second time I was away from keyboard doing nothing and came back to the blue screen in progress.

I never get this with any other software, so something must be going on with Eco to trigger it. However, it doesn't seem to be caused by any specific in game action.

System info:
OS: Windows 10 Professional 64-bit
Processor: Intel Core i7-5820k CPU @ 3.30 Ghz (water cooled)
GPU: Nvidia 980 Ti (water cooled)
RAM: 64.0 GB

**Comment by GophTheGreat** [comment]
I'm going to second Xeanoa and say that sounds like a symptom of a deeper problem.

**Comment by Elliander** [comment]
Well, when my computer was put together it went through stress testing that never triggered that BSOD error. Additionally, I have run Eco for days without this error and also had it happen twice in the same day, regardless of how hard the CPU is working.

Can't be an issue with my power supply either, because in addition to being rated for the power draw I use a voltage regulator and a high end UPS for over an hour at full load without power available and without problems.

Is there a piece of software that I could run which should consistently trigger this crash if it is a problem with my CPU?

**Comment by metachronism** [comment]
@Elliander can you give 7.4 a try, to see if this issue is any better?  Eco is a big hog on resources, so its not surprising that it might cause instability on some hardware that isnt usually stressed by other programs

**Comment by Elliander** [comment]
I am running 0.7.4.0-beta-staging-b8e0bc58

As far as resource use goes, I had the task manager open the entire time playing yesterday (and going out of my way to try to stress it) and found that the usage for the Eco process was never higher than 33% and rarely above 20% - running at 4.60 GHz apparently. It seemed to use more processing when mining lots of rocks at once, which made sense.

After a while I put videos up while playing and the monitor went over 50%, but still nothing.

While I did not get any BSOD yesterday (so don't know yet what the levels there should be), but did have two crashes. Although, both occurred after running the game for far longer than I had before. I wanted to see if processor usage went up over time, which could indicate a separate problem, but it didn't. (although I did find multiple new bugs to report)

![eco_crash](https://user-images.githubusercontent.com/15022396/38907830-31619c08-4283-11e8-8407-4de81bac934b.jpg)

Is there a place where crash reports should be stored? Not finding them.
